rest_framework/fields.py

```diff
@@ -53,14 +53,30 @@ def is_simple_callable(obj):
     """
     True if the object is a callable that takes no arguments.
     """
+    if not hasattr(inspect, 'signature'):
+        return py2k_is_simple_callable(obj)
```

Comment:
Could we branch by version and have two differing definations of `is_simple_callable`? That'd match the style we use elsewhere in this module.